samples/dfu/nrf5340/Kconfig

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2022 INTELLINIUM <giuliano.franchetto@intellinium.com>
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "LTE UART DFU sample"
```

Comment:
The problem with this is it seems to be based around the nrf5340dk, but using LTE. There is no nrf9160 on the nrf5340dk, so I don't see it being much use if people can't actually use it. If it used bluetooth then that would be acceptable.